### ISSUE TITLE
Error if NPC has more than 1 available quest

### ISSUE THREAD

**Issue by StKillReal** [PikaMug/Quests#2102]
### What is your Quests version / build number (do *not* say "latest")?

![image](https://user-images.githubusercontent.com/60330193/237018432-1c2a69f9-f616-4ebc-82dd-74f7250ec5d0.png)

### Describe the problem and how to replicate it.

If NPC has 1 quest, I can get it without any problems, but if he has more than 1 I only can see yellow error text. NPC on the first screenshot has 2 quests, on the second screen NPC has only 1 quest
![image](https://user-images.githubusercontent.com/60330193/237018874-308082d3-5f73-4820-814c-3eba2dd76aa5.png)
![image](https://user-images.githubusercontent.com/60330193/237018959-636e4a7a-6fc3-483e-85af-b84782dee2ea.png)
 

### Any console errors to share?

```shell
Idk if it because of Quests or not, but anyway
https://pastebin.com/jD2JupqD
```

### If applicable, can you provide an example quest from quests.yml?

```yaml
custom2:
    name: В погоне за пшеницей
    ask-message: Приветствую! Я занимаюсь тем, что кормлю всех этих идио... Всех жителей
      этого города в общем. С тех пор как эти глыбы земли поднялись наверх, делать
      это стало почти невозможно. Ты не мог бы отнести хлеб всем жителям города? Я
      заплачу, не переживай.
    finish-message: Спасибо! Возвращайся через несколько дней и можешь снова мне помочь.
    npc-giver-uuid: cfb37150-a67c-4394-8f42-592e0cf0c445
    stages:
      ordered:
        '1':
          items-to-craft:
          - ==: org.bukkit.inventory.ItemStack
            v: 3337
            type: BREAD
            amount: 64
          start-message: Хлеб мы пока сделать не успели, так что было бы здорово,
            если бы ты смог есть раздобыть.
          complete-message: Отлично, хлеб у меня есть. осталось найти всех жителей..
        '2':
          items-to-deliver:
          - ==: org.bukkit.inventory.ItemStack
            v: 3337
            type: BREAD
            amount: 8
          npc-delivery-uuids:
          - e0a98829-cf96-4ea0-9c6e-c14d3a9684f2
          delivery-messages:
          - Спасибо, но обещали же 8 булок, так что жду остальные.
          complete-message: Так, там же в казино бармен со стойки отойти не может!
        '3':
          items-to-deliver:
          - ==: org.bukkit.inventory.ItemStack
            v: 3337
            type: BREAD
            amount: 8
          npc-delivery-uuids:
          - 6d5d3df9-ddee-4e21-b3db-683c4bad73aa
          delivery-messages:
          - Спасибо, но обещали 8 булок, так что жду еще.
          complete-message: Спасибо за хлеб, дружище! Слушай, я знаю одного парня,
            он сидит в подземелье в конце пшеничного поля. Мы с ним давние друзья..
            Не отнесешь ему хлеба?
        '4':
          items-to-deliver:
          - ==: org.bukkit.inventory.ItemStack
            v: 3337
            type: BREAD
            amount: 8
          npc-delivery-uuids:
          - 4d403453-0ab6-4a7b-b1cb-20f4e0f26b22
          delivery-messages:
          - Тут так мало?
          complete-message: Это бармен тебя прислал? Спасибо... Слушай, у меня тут
            есть одно дельце, как закончишь с хлебом, заходи. Обсудим. У тебя же еще
            остался хлеб? Не отнесешь моей знакомой на верхний остров?
        '5':
          items-to-deliver:
          - ==: org.bukkit.inventory.ItemStack
            v: 3337
            type: BREAD
            amount: 8
          npc-delivery-uuids:
          - 62c7cb55-0075-4817-9110-76e22ff746cf
          delivery-messages:
          - Как-то маловато хлеба для недельного ожидания...
          complete-message: Спасибо, незнакомец. Я смотрю ты разносишь хлеб сегодня?
            В таком случае не занесешь моему мужу? Он сейчас должен быть в кузне возле
            гладиаторской арены.
        '6':
          locations-to-reach:
          - world 558.0 62.0 -34.0
          reach-location-radii:
          - 5
          reach-location-names:
          - Кузня возле PvP-арены
          complete-message: Странно, его тут нет, но я чувствую какой-то приятный
            аромат... Нужно проверить куда он идет.
        '7':
          locations-to-reach:
          - world 496.0 73.0 -42.0
          reach-location-radii:
          - 5
          reach-location-names:
          - Вход в один из домов
          complete-message: Это определенно он. Поднимусь выше
        '8':
          locations-to-reach:
          - world 493.0 80.0 -40.0
          reach-location-radii:
          - 3
          reach-location-names:
          - Спальня на 2 этаже
          complete-message: Черт, ты кто еще такой?!
        '9':
          items-to-deliver:
          - ==: org.bukkit.inventory.ItemStack
            v: 3337
            type: BREAD
            amount: 8
          npc-delivery-uuids:
          - 2c9fdec7-8a24-4463-8099-f8b67fe39e93
          delivery-messages:
          - '*Ему нужно было 8 хлеба*'
          complete-message: Черт, это конечно здорово.. Я так понимаю тебя прислала
            моя жена? Неловко, конечно... Слушай, давай мы просто замнем этот случай,
            а я помогу тебе заработать денег. Обращайся, как будешь свободен. Только
            не говори ей, прошу.
        '10':
          items-to-deliver:
          - ==: org.bukkit.inventory.ItemStack
            v: 3337
            type: BREAD
            amount: 8
          npc-delivery-uuids:
          - 4488f371-f88a-4172-bf32-a553af3f3268
          delivery-messages:
          - Так мало?!
          start-message: Да уж... Пойду поищу людей в тверне, там уж точно кто-то
            должен быть.
          complete-message: Спасибо, дружище! Заходи к нам в таверну, тут бывает довольно
            весело!
        '11':
          items-to-deliver:
          - ==: org.bukkit.inventory.ItemStack
            v: 3337
            type: BREAD
            amount: 16
          npc-delivery-uuids:
          - 11ef8578-ea2f-46b8-a86a-fdbb7a92a3db
          delivery-messages:
          - Я просил 16 булок
          start-message: Остался рыбак...
          complete-message: Спасибо огромное! Я хоть его сам и не ем, но это отличная
            приманка. Именно поэтому мне и нужен двойной объем.
        '12':
          npc-uuids-to-talk-to:
          - cfb37150-a67c-4394-8f42-592e0cf0c445
    rewards:
      money: 1000
      quest-points: 1
    planner:
      cooldown: 172800
      override: false
    options:
      allow-commands: true
      allow-quitting: true
      ignore-silk-touch: true
      use-parties-plugin: true
      share-progress-level: 1
      same-quest-only: true
      share-distance: 0.0
      handle-offline-players: false
      ignore-block-replace: true
```

**Comment by PikaMug** [comment]
Duplicate of #2091

**Comment by StKillReal** [comment]
> Duplicate of #2091

It's not duplicate, it's another issue. How u can see I use build №406, but u told that issue was fixed in build №405
![image](https://github.com/PikaMug/Quests/assets/60330193/0a9cd6b7-4afc-4222-9f75-c91fc6bdbba9)

**Comment by StKillReal** [comment]
When I try to take quest using /questadmin give I have an error in console. Error appear for only one quest and idk if it have anything common with my issue. (quest still work even with this error)
I've tried to create new NPC and few new quests, but when NPC have 2 quests it's stop to work
![image](https://github.com/PikaMug/Quests/assets/60330193/5e3e95d5-8e07-4786-9dfa-d87fc6eaa4d7)

**Comment by PikaMug** [comment]
@StKillReal Your last error is unrelated and was already fixed in #2103

This issue should be resolved in https://ci.codemc.io/job/PikaMug/job/Quests/414/
